students/WesleyM/Lesson3/StringFormatLab.py

```diff
@@ -0,0 +1,52 @@
+def task_one():
+    """Converts (2, 123.4567, 10000, 12345.67) to 'file_002 :   123.46, 1.00e+04, 1.23e+04'"""
+    string = (2, 123.4567, 10000, 12345.67)
+    print("file_{:03d}: {:.2f}, {:.2e}, {:.2e}".format(string[0], string[1],string[2],string[3]))
+    return
+
+def task_two():
+    """Converts (2, 123.4567, 10000, 12345.67) to 'file_002 :   123.46, 1.00e+04, 1.23e+04'using f-strings"""
+    string = (2, 123.4567, 10000, 12345.67)
+    print(f"file_{string[0]:03d}: {string[1]:.2f}, {string[2]:.2e}, {string[3]:.2e}")
+    return
+    
+def task_three(in_tuple):
+    """Takes a user input tuple and writes in the form of 'the X numbers are: a, b, c'"""
+    l = len(in_tuple)
```

Comment:
Nice work! It would be a bit more readable to just use the call to `len()` in your code, rather than assigning to `l`:

`form_string = ("The {} numbers are: " + ", ".join(["{}"] * len(in_tuple))).format(len(in_tuple), *in_tuple)`